Gruntfile.js

```diff
@@ -166,6 +168,14 @@ module.exports = function (grunt) {
                 }
             }
         },
+        postcss: {
+            options: {
+                processors: [
+                  autoprefixer({ browsers: ["Explorer >= 10", "Firefox >= 26", "Chrome >= 31", "Safari >= 7", "Opera >= 19", "iOS >= 3.2", "Android >= 2.1"] }).postcss
```

Comment:
This has jshint indent issues.  Also, it should be:

```js
+        postcss: {
+            options: {
+                processors: [
+                    autoprefixer({
                          browsers: [
                              "Explorer >= 10",
                              "Firefox >= 26",
                              "Chrome >= 31",
                              "Safari >= 7",
                              "Opera >= 19",
                              "iOS >= 3.2",
                              "Android >= 2.1"
                           ]
                       }).postcss
```

Also, change the Android version to `>= 4.4`